dbt/adapters/snowflake/connections.py

```diff
@@ -143,6 +144,12 @@ def __post_init__(self):
 
         self.account = self.account.replace("_", "-")
 
+        # only default `reuse_connections` to `True` if the user has not turned on `client_session_keep_alive`
+        # having both of these set to `True` could lead to hanging open connections, so it should be opt-in behavior
+        if self.client_session_keep_alive is False and self.reuse_connections is None:
+            print("Updating reuse_connections")
```

Comment:
Do you want to keep the print here?
If you want to log something, look into `fire_event(Note(...))`.

Reply:
lol, no, good catch, I'll update it